src/metaObject.cxx

```diff
@@ -1403,7 +1403,7 @@ M_SetupWriteFields(void)
       if(m_WriteCompressedDataSize && m_CompressedDataSize>0)
         {
         mF = new MET_FieldRecordType;
-        MET_InitWriteField(mF, "CompressedDataSize", MET_UINT,
+        MET_InitWriteField(mF, "CompressedDataSize", MET_ULONG_LONG,
```

Comment:
yes there might be - as I have stated in the pull request text above, MET_ULONG_LONG previously were written without the name field before them. I have no idea whether this is a bug and this has not turned up yet because MET_ULONG_LONG fields are not used anywhere else, or whether not printing the name is actually required somewhere... I only used the MetaIO library for reading/writing Volume Data MetaImages so far, I hope someone else could point out whether this might have side effects for other data!